plugin/pkg/auth/authorizer/node/graph_populator.go

```diff
@@ -0,0 +1,41 @@
+package node
+
+import (
+	"k8s.io/client-go/tools/cache"
+
+	coreinformers "k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/core/internalversion"
+)
+
+type GraphPopulator struct {
+	graph *Graph
+
+	podInformer              cache.SharedIndexInformer
```

Comment:
We generally haven't been storing refs to the underlying `cache.SharedIndexInformer` objects. If you're using a shared informer, it's generally safe to assume that something else (e.g. kube-controller-manager) will start it.